assets/scss/base/_table.scss

```diff
@@ -104,6 +104,11 @@ table {
     font-weight: 400;
   }
 }
+
+.table--heavy-header {
+    border-bottom: em(2) solid $govuk-blue-colour;
+}
```

Comment:
I like the:
> .table--header-highlight {
>    th {
>       border-bottom: em(2) solid $govuk-blue-colour;
>    }
> }

but shouldn't we try to keep the class as close as possible to the component that is being affected i.e. the th?

So it would be .table__header--highlight